packages/core/src/hosted-form/hosted-form.ts

```diff
@@ -68,14 +67,24 @@ export default class HostedForm {
 
     async submit(payload: OrderPaymentRequestBody): Promise<void> {
         try {
-            return await this._getFirstField().submitForm(
+            const {payload: eventPayload} = await this._getFirstField().submitForm(
                 this._fields.map(field => field.getType()),
                 this._payloadTransformer.transform(payload)
             );
+
+            const { response, data } = eventPayload;
+
+            if (data.paymentMethod?.type === PaymentStrategyType.PPSDK) {
+                await this._ppsdkStepHandler.handle(
+                    response,
+                    this._paymentHumanVerificationHandler,
```

Comment:
Avoid injecting this via `StepHandler#handle` method as you want its parameters to apply to all kinds of handler, not just human verification.